Test/Publish-WhiskeyBuildMasterPackage.Tests.ps1

```diff
@@ -255,16 +333,59 @@ function ThenTaskFails
     }
 }
 
-Describe 'Publish-WhiskeyBuildMasterPackage.when called' {
+Describe 'Publish-WhiskeyBuildMasterPackage.when current branch is mapped to release' {
+    Init
+    GivenProperty $defaultPackageVariable
+    WhenCreatingPackage
+    ThenCreatedPackage '9.8.3' -InRelease 'release' -ForApplication 'application' -AtUri 'https://buildmaster.example.com' -UsingApiKey 'fubarsnafu' -WithVariables @{ One = 'Two'; Three = 'Four' }
+    ThenPackageDeployed -AtUri 'https://buildmaster.example.com' -UsingApiKey 'fubarsnafu'
+}
+
+Describe 'Publish-WhiskeyBuildMasterPackage.when branch wildcard is mapped to release' {
+    Init
+    GivenDeployTo @{ 'BranchName' = 'dev*'; 'ReleaseName' = $releaseName }
+    GivenProperty $defaultPackageVariable
+    WhenCreatingPackage
+    ThenCreatedPackage '9.8.3' -InRelease 'release' -ForApplication 'application' -AtUri 'https://buildmaster.example.com' -UsingApiKey 'fubarsnafu' -WithVariables @{ One = 'Two'; Three = 'Four' }
+    ThenPackageDeployed -AtUri 'https://buildmaster.example.com' -UsingApiKey 'fubarsnafu'
+}
+
+Describe 'Publish-WhiskeyBuildMasterPackage.when current branch is one of multiple mapped to release' {
     Init
-    GivenProperty @{
-                        'PackageVariable' = @{
-                                                    'One' = 'Two';
-                                                    'Three' = 'Four';
-                                             }
-                   }
+    GivenDeployTo @( @{ 'BranchName' = 'default' ; 'ReleaseName' = $releaseName }, @{ 'BranchName' = @( 'release', 'feature/*', 'develop' ); 'ReleaseName' = $releaseName } )

```

Comment:
Use a `Given` pre-condition to specify the current branch.

Reply:
Added